src/Tests/TransactionTest.php

```diff
@@ -106,324 +90,6 @@ public function test_method_get_id()
         );
     }
 
-    public function test_additional_property_merchant_id()
```

Comment:
A lot of test deleted. Are they not needed at all? Will the percentage of code coverage by tests decrease?
It's nice that new additional property filter will check it, and completely replaces these tests